### PR TITLE
Copyedited the tutorial

### PR DESCRIPTION
**Pull request by Sayanta66**
Copyedited the tutorial. Fixed all of the document errors and punctuation errors along with typos

## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: docs/tutorials/add-environment-variable.md
Comment (adityaoberai) [inline docs/tutorials/add-environment-variable.md:16]: In the first edit,

> First of all, we add the new

`add` is the correct usage here. This is because the line intends to share about what should be done rather than what has been done.

---

In the second edit, 

> Copy the existing variables description to create a new one

I believe the right correction would be `existing variable description` rather than `description of the existing variable` (as *variable description* is a term of its own)

File: docs/tutorials/add-environment-variable.md
Comment (adityaoberai) [inline docs/tutorials/add-environment-variable.md:12]: It would be great if you could correct the `variable` spelling here as well :)

File: docs/tutorials/add-environment-variable.md
Comment (adityaoberai) [inline docs/tutorials/add-environment-variable.md:23]: Docker Compose as a term here would not be hyphenated.

At the max, I would suggest editing `docker compose` to `Docker Compose`

File: docs/tutorials/add-environment-variable.md
Comment (adityaoberai) [inline docs/tutorials/add-environment-variable.md:26]: > The Appwrite maintainers use the `docker-compose.yml` file during development
 
 The first edit here is actually not necessary and create an inconsistency in the voice of the verbs used throughout the line. It would be a good idea to revert back to the original here:
 
 > The `docker-compose.yml` file is used by the Appwrite maintainers during development

File: docs/tutorials/add-environment-variable.md
Comment (adityaoberai) [inline docs/tutorials/add-environment-variable.md:32]: I recommend replacing the article `the` with `a`

**Review by adityaoberai — CHANGES_REQUESTED** [review]
Thanks for the edits @Sayanta66. I have requested a few changes that I would like you to look at once. Let me know here if you have any queries about them :)

**Comment by adityaoberai** [comment]
Apart from the edits @Sayanta66, it would be great if you create your Pull Requests using the template we have in the future. An example you can view to understand how to use the PR template for a documentation edit is #1561

**Comment by Sayanta66** [comment]
Thanks @adityaoberai ...I will do that from the future

File: docs/tutorials/add-environment-variable.md
Comment (Sayanta66) [reply docs/tutorials/add-environment-variable.md:26]: Actually using active voice is much better in a docs compared to passive voice. If you allow me, I will copy-edit the entire document to active voice.

File: docs/tutorials/add-environment-variable.md
Comment (Sayanta66) [reply docs/tutorials/add-environment-variable.md:23]: Okay yeah! I will do this.

File: docs/tutorials/add-environment-variable.md
Comment (adityaoberai) [reply docs/tutorials/add-environment-variable.md]: That will not be necessary at the moment. For now, let's keep that sentence consistent with passive voice.

**Comment by eldadfux** [comment]
@adityaoberai is this good for merge?

File: docs/tutorials/add-environment-variable.md
Comment (adityaoberai) [inline docs/tutorials/add-environment-variable.md:12]:
```suggestion
### Naming environment variable
```

File: docs/tutorials/add-environment-variable.md
Comment (adityaoberai) [inline docs/tutorials/add-environment-variable.md:16]:
```suggestion
First of all, we add the new environment variable to `app/config/variables.php` in the designated category. If none of the categories fit, add it to the General category. Copy the existing variable description to create a new one so that you will not miss any required fields.
```

We don't use the past tense here for `add` as this is a general instruction

**Review by adityaoberai — CHANGES_REQUESTED** [review]
@Sayanta66 I have requested a few more changes, after which this PR should be good to merge 👍

**Comment by Sayanta66** [comment]
@adityaoberai committed the changes

**Comment by eldadfux** [comment]
@Sayanta66 all changes merged! Thanks for contributing!

**Pull request merged**
